homeassistant/components/group.py

```diff
@@ -145,7 +145,7 @@ def setup(hass, config):
     """Setup all groups found definded in the configuration."""
     for object_id, conf in config.get(DOMAIN, {}).items():
         name = conf.get(CONF_NAME, object_id)
-        entity_ids = conf[CONF_ENTITIES]
+        entity_ids = conf.get(CONF_ENTITIES, [])
```

Comment:
This will not work if the value is `None`:
```
>>> {"hello" : None}.get("hello", 5) is None
True
```

This could work:
```python
entity_ids = conf.get(CONF_ENTITIES) or []
```